src/App.jsx

```diff
@@ -1,56 +1,128 @@
-import { initialColors } from "./lib/colors";
+import { initialThemes, initialColors } from "./lib/colors";
```

Comment:
Here you can remove initialColors/Like me/ from lib component bcz  they are defined in initial themes:
`import { initialThemes } from "./lib/colors"; `